connectors/spark/etc/conf/server.properties

```diff
@@ -1,6 +1,10 @@
 server.env=dev
 ## temp credential config for s3 (Multiple s3 config can be added by incrementing the index)
```

Comment:
@ravivj-db I really think that server.properties should be loadable from the classpath as well. this is not a good set up. this file should be spark/test/resources

Reply:
ACK.